### PR TITLE
Add Collision Prevention docs

### PR DESCRIPTION
**Pull request by thetechnick**

### CODE REVIEW

File: content/en/docs/concepts/collision-prevention.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- **IfNoController**  
+  Allows operations on objects without an owner reference. This includes objects created directly by users, but excludes objects created via controllers.
+- **None**  
+  Disables all collision prevention checks.  
+  *Only use this settings when you know what you are doing!*
+
+### Package
+
+To change the collision protection settings for objects within a package, use the: 
+`package-operator.run/collision-protection` annotation.
```
Comment (Nanyte25) [inline content/en/docs/concepts/collision-prevention.md]: An example of the `oc annotate --overwrite` command might be useful here

File: content/en/docs/getting_started/api-reference.md [hunk 2]
```diff
@@ -1288,39 +1315,54 @@ spec:
...
 | `metadata` <br>metav1.ObjectMeta |  |
-| `spec` <br><a href="#packagemanifestspec">PackageManifestSpec</a> | PackageManifestSpec represents the spec of the packagemanifest containing the details about phases and availability probes. |
+| `spec` <br><a href="#packagemanifestspec">PackageManifestSpec</a> | PackageManifestSpec represents the spec of the packagemanifest containing the<br>details about phases and availability probes. |
 | `test` <br><a href="#packagemanifesttest">PackageManifestTest</a> | PackageManifestTest configures test cases. |
 
 
 ### PackageManifestLock
 
-
+PackageManifestLock allows locking packages to specific versions.
```
Comment (Nanyte25) [inline content/en/docs/getting_started/api-reference.md:1365]: It might be useful to give a version lock example here.

Comment (thetechnick) [reply content/en/docs/getting_started/api-reference.md:1365]: An auto-generated example begins just a few lines below :+1:

File: content/en/docs/concepts/collision-prevention.md (hunk 1 above)
Comment (thetechnick) [reply content/en/docs/concepts/collision-prevention.md]: Expanded the sentence to clarify, the annotation is not set on an object in the cluster, it's set within the Package filesystem. The oc command won't work here.

**Pull request merged**